geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/concurrent/ConcurrentParallelGatewaySenderOperation_2_DUnitTest.java

```diff
@@ -55,70 +55,6 @@ protected final void postSetUpWANTestBase() throws Exception {
     IgnoredException.addIgnoredException("Unexpected IOException");
   }
 
-  @Test
```

Comment:
Where in the docs is this not supported?  I see a caution but I do know some people do use this feature (which is what lead to discovery and fix for the issue where secondary queues would continue to grow).

Whether or not it's supported, the test currently passes on develop, if this test is not passing with this diff, the diff negatively impacts the test in some way.  Do we have an understanding of why the test is hanging now?

Reply:
It looks like the close() implementation causes this to fail.

Reply:
If my understanding is correct, according to [https://geode.apache.org/docs/guide/12/topologies_and_comm/topology_concepts/multisite_overview.html](url) , in chapter Stopping Gateway Senders it is stated that:
> The API and gfsh command stops the parallel gateway sender in one member, which causes data loss because events to buckets in that member will be dropped by the stopped sender. The partitioned region does not failover in this scenario since the member is still running. Instead, to ensure that the remaining events are sent, shut down the entire member to ensure proper failover of partition region events.

Reply:
I think it's describing the effects of calling these api's directly (which one would assume would mean, not supported/recommended).  However I do know someone has used these api's directly in the past. 

In either case, the close() implementation is causing replication to the other wan site not to occur.  I have some other private tests running that show weird symptoms with this PR that I'll spend some time on.  I'll try to find out why this is not replicating across the wan with this PR and provide more detail.

Reply:
Thanks

Reply:
Hi Mario, 
The close in combination with stopping gateways senders can cause odd issues, like PartitionedOfflineExceptions, RegionDestroyedExceptions or behavior like this test is exhibiting.  We have some internal applications that are running into these types of issues with this diff.  Other than our own internal applications I do know of some users that rely on being able to do this.  A change of this behavior should probably have a discuss on the dev list about current vs future behavior and ramifications. Thanks!

Reply:
Solution with closing of region is removed, so there will be no odd issues, and weird symptoms.